plugins/inputs/nfsclient/nfsclient.go

```diff
@@ -190,6 +190,10 @@ func (n *NFSClient) parseStat(mountpoint string, export string, version string,
 		fields["bytes"] = nline[3] + nline[4]
 		fields["rtt"] = nline[6]
 		fields["exe"] = nline[7]
+		fields["rtt_per_op"] = 0.0
+		if nline[0] > 0 {
+			fields["rtt_per_op"] = float64(nline[6]) / float64(nline[0])
```

Comment:
rtt_per_op is just rtt/ops, right? This is the kind of thing processors are meant to handle. Why not use starlark or something to do this?

Reply:
If someone is using the NFS client input plugin, they are probably monitoring their system and are interested in a number of key metrics. 

Similarly, in the memory input plugin we calculate used and available percents, in the CPU plugin we calculate all sorts of usage percentages, and in the LVM plugins, we calculate used percentages as well. I feel that the NFS client, knowing your average RTT falls into this same category of something pretty much everyone is going to want to do. Asking every user to do this calculation, when we could add it seems to not be the right user experience.

I can understand not wanting Telegraf to calculate lots of different averages and other metrics, but this seems to fall in line with other monitoring and reporting of key metrics that make it hard not to have it included.